webapp-django/crashstats/settings/local.py-dist

```diff
@@ -168,3 +168,43 @@ SECRET_KEY = 'you must change this'
 #           'component=General&groups=client-services-security',
 #    'label': 'File a bug in bugzilla'
 #}
+
+
+# Dataservice API configuration
+# Extend dataservices settings from settings/base.py here
+# At a minimum, you'll probably want to change db username/password. All
+# dataservice objects inherit resource configuration and so can all
+# have their database resource configuration set once in 'secrets.postgresql'
+# and 'resource.postgresql' keys.
+# Here's an example:
+#from . import base
+#try:
+#    DATASERVICE_CONFIG_BASE = base.DATASERVICE_CONFIG_BASE
+#except AttributeError:
+#    pass
```

Comment:
Why would why they have a `local.py-dist` that is more up to date than
`settings/base.py`?!


On Mon, Jan 12, 2015 at 2:30 PM, Selena Deckelmann <notifications@github.com
> wrote:

> In webapp-django/crashstats/settings/local.py-dist
> <https://github.com/mozilla/socorro/pull/2506#discussion-diff-22828498>:
>
> > @@ -168,3 +168,43 @@ SECRET_KEY = 'you must change this'
> >  #           'component=General&groups=client-services-security',
> >  #    'label': 'File a bug in bugzilla'
> >  #}
> > +
> > +
> > +# Dataservice API configuration
> > +# Extend dataservices settings from settings/base.py here
> > +# At a minimum, you'll probably want to change db username/password. All
> > +# dataservice objects inherit resource configuration and so can all
> > +# have their database resource configuration set once in 'secrets.postgresql'
> > +# and 'resource.postgresql' keys.
> > +# Here's an example:
> > +#from . import base
> > +#try:
> > +#    DATASERVICE_CONFIG_BASE = base.DATASERVICE_CONFIG_BASE
> > +#except AttributeError:
> > +#    pass
>
> If someone was deploying this change and upgrading from a previous version
> of socorro, and they pushed their config changes before updating their app
> -- without this try/except block our config would break their app.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/mozilla/socorro/pull/2506/files#r22828498>.
>



-- 
Peter Bengtsson
Mozilla Web Engineering